packages/moonstone/LabeledItem/LabeledItem.js

```diff
@@ -96,7 +96,7 @@ const LabeledItemBase = kind({
 	},
 
 	render: ({children, css, disabled, label, titleIcon, ...rest}) => (
-		<Controller disabled={disabled} {...rest} css={css}>
+		<Controller disabled={disabled} data-webos-voice-intent="Select" {...rest} css={css}>
```

Comment:
This isn't needed. `Item` already has the prop set.

Reply:
I will remove it.